tests/unit/adapter_tests.py

```diff
@@ -0,0 +1,66 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright © 2016 IBM Corp. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+from cloudant.client import CouchDB
+from cloudant.adapters import Replay429Adapter
+from requests.packages.urllib3.util import Retry
+from .unit_t_db_base import UnitTestDbBase
+
+class AdapterTests(UnitTestDbBase):
+    """
+    Unit tests for transport adapters
+    """
+
+    def test_new_Replay429Adapter(self):
```

Comment:
It was basically to demonstrate that the different options are usable in the constructor as a protection against regressing the init.
The values get passed into requests/urllib3 so I didn't assert anything on them because it would require knowing something about where they end up in those libs.